src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag

```diff
@@ -20,7 +29,7 @@
     <div id="visibility-options${commentIdx}" class="panel panel-default"
         style="display: none;">
         <div class="panel-heading">Visibility Options</div>
-        <table class="table text-center text-color-black">
+        <table class="table text-center" style="color: #000;">
```

Comment:
Yeah, this is the one that's different compared to others. FRC used `style="color: #000"` as well. We can settle this is another issue I guess?